docs/src/topics/boundary_conditions.md

```diff
@@ -93,6 +93,21 @@ end
     Most examples make use of Dirichlet boundary conditions, for example [Heat
     Equation](@ref tutorial-heat-equation).
 
+## L2ProjectedDirichlet
+Some interpolations don't have nodal support points:
+``H(\mathrm{curl})`` interpolations, e.g., `Nedelec`, are associated to edges
+and ``H(\mathrm{div})`` interpolations, e.g. `RaviartThomas`, are associated to facets.
+While normal `Dirichlet` boundary conditions assume the existence of such nodal support points,
+Ferrite provides the `L2ProjectedDirichlet`, which instead finds the degree of freedom values that
+minimizes the L2-distance between the prescribed function, ``f(\boldsymbol{x},t,\boldsymbol{n})``,
+and the finite element interpolation space. Although standard interpolations are not currently supported,
+the figure below illustrates well the difference between applying a standard `Dirichlet` condition and an
+`L2ProjectedDirichlet` condition when the prescribed function cannot be described by the chosen FE-interpolation.
+
+![L2ProjectedDirichlet illustration](https://raw.githubusercontent.com/Ferrite-FEM/Ferrite.jl/gh-pages/assets/L2ProjectedDirichlet.svg)
```

Comment:
This should be queried in the asset download script and reference the local figure, right?

Reply:
Not sure what the best approach here is. Currently, only assets are downloaded for the tutorials, but I guess it makes sense to download it instead. Adding an svg to source control really bloats the number of lines etc, so not so nice.

Perhaps possible to use something like
```@example
using Downloads: download #hide
download(joinpath(@__DIR__, L2ProjectedDirichlet.svg), "https://raw.githubusercontent.com/Ferrite-FEM/Ferrite.jl/gh-pages/assets/L2ProjectedDirichlet.svg") #hide
nothing #hide
```
![L2ProjectedDirichlet illustration](L2ProjectedDirichlet.svg)

With #1037 would aid this by centralizing the asset location.

@fredrikekre: What are your views on how we should manage such assets in the non-tutorial part of the docs?

Reply:
> Adding an svg to source control [...]

Do we have git lfs configured? If so, then we can just add it via git lfs to treat it similar to binaries.

Reply:
Downloaded it and added it to a new `downloaded_assets` directory which is ignored by git.